src/nanoemoji/glue_together.py

```diff
@@ -0,0 +1,37 @@
+from fontTools.ttLib.tables import otTables as ot
+from fontTools import ttLib
+import os
+
+_ORIGINAL="/tmp/Bungee7.ttf"
+_COLR="/tmp/bungee7/AnEmojiFamily.ttf"
+
+font = ttLib.TTFont(_ORIGINAL)
+colr = ttLib.TTFont(_COLR)
+_OUTPUT = '/tmp/bungee7/Bungee_SVG_COLR.ttf'
+
+# Copy all glyphs used by COLR over
+_glyphs_to_copy = set()
+
+def _collect_glyphs(paint):
+  if paint.Format == ot.PaintFormat.PaintGlyph:
+    _glyphs_to_copy.add(paint.Glyph)
+
+for record in colr["COLR"].table.BaseGlyphList.BaseGlyphPaintRecord:
+  record.Paint.traverse(colr["COLR"].table, _collect_glyphs)
+
+for glyph_name in _glyphs_to_copy:
+  print("Copy glyph", glyph_name)
+  font['glyf'].glyphs[glyph_name] = colr['glyf'].glyphs[glyph_name]
+  font['hmtx'].metrics[glyph_name] = colr['hmtx'].metrics[glyph_name]
+  font.getGlyphOrder().append(glyph_name)
```

Comment:
even if `font.getGlyphOrder().append(...)` works right now, I would not rely on getGlyphrOrder continuing to return a mutable list in the future; there were discussions to have it return a list copy or a tuple even. The documented API is to call setGlyphOrder at the end when you have your new list.

Reply:
Sorry, I was apparently in lazy idiot mode. Will fix :)

Reply:
Moving to setGlyphOrder ias causing a crash so I'm going back on that change until I have a chance to poke at that.

```
Traceback (most recent call last):
  File "/usr/lib/python3.9/runpy.py", line 197, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "/usr/lib/python3.9/runpy.py", line 87, in _run_code
    exec(code, run_globals)
  File "glue_together.py", line 76, in <module>
    app.run(main)
  File "venv/lib/python3.9/site-packages/absl/app.py", line 312, in run
    _run_main(main, args)
  File "venv/lib/python3.9/site-packages/absl/app.py", line 258, in _run_main
    sys.exit(main(argv))
  File "glue_together.py", line 67, in main
    target.save(FLAGS.output_file)
  File "venv/lib/python3.9/site-packages/fontTools/ttLib/ttFont.py", line 181, in save
    writer_reordersTables = self._save(tmp)
  File "venv/lib/python3.9/site-packages/fontTools/ttLib/ttFont.py", line 221, in _save
    self._writeTable(tag, writer, done, tableCache)
  File "venv/lib/python3.9/site-packages/fontTools/ttLib/ttFont.py", line 630, in _writeTable
    self._writeTable(masterTable, writer, done, tableCache)
  File "venv/lib/python3.9/site-packages/fontTools/ttLib/ttFont.py", line 634, in _writeTable
    tabledata = self.getTableData(tag)
  File "venv/lib/python3.9/site-packages/fontTools/ttLib/ttFont.py", line 656, in getTableData
    return self.tables[tag].compile(self)
  File "venv/lib/python3.9/site-packages/fontTools/ttLib/tables/_m_a_x_p.py", line 43, in compile
    self.recalc(ttFont)
  File "venv/lib/python3.9/site-packages/fontTools/ttLib/tables/_m_a_x_p.py", line 62, in recalc
    self.numGlyphs = len(glyfTable)
  File "venv/lib/python3.9/site-packages/fontTools/ttLib/tables/_g_l_y_f.py", line 306, in __len__
    assert len(self.glyphOrder) == len(self.glyphs)
AssertionError
```

Reply:
...oh am idiot, never mind lol

Reply:
Curiously I had to set glyph order on both the TTFont and the glyf table.